packages/ui/src/components/TimeInputV2/index.tsx

```diff
@@ -157,7 +157,7 @@ padding-inline: ${({ theme }) => theme.space['0.25']};
 
 type TimeInputProps = {
   placeholder?: Time
-  value?: Date
+  value?: Date | string | null
```

Comment:
I'm confused why we need to allow string while you could provide `value={new Date(value)}` while calling the component? I prefer if the types are strict and the manipulation is happening outside of the component as a string could be anything including a real date.